js/src/components/free-listings/setup-free-listings/form-content.js

```diff
@@ -22,13 +22,16 @@ import ConditionalSection from '.~/components/conditional-section';
  *
  * @param {Object} props React props.
  * @param {string} [props.submitLabel="Complete setup"] Submit button label.
+ * @param {boolean} [props.hideTaxRates=false] Whether to hide tax rate section.
```

Comment:
This JSDoc needs to be updated as well.

```suggestion
 * @param {boolean} props.hideTaxRates Whether to hide tax rate section.
```